components/ws-manager-mk2/controllers/create.go

```diff
@@ -447,7 +447,7 @@ func createWorkspaceContainer(sctx *startWorkspaceContext) (*corev1.Container, e
 			PeriodSeconds:       1,
 			SuccessThreshold:    1,
 			TimeoutSeconds:      1,
-			InitialDelaySeconds: 3,
+			InitialDelaySeconds: 0,
```

Comment:
This is unrealistic, even in [ws-manager, we set a value of 2 seconds](https://github.com/gitpod-io/gitpod/blob/6a8b9a9971d4d700b42a1cbca5e0e71a42efbe0d/components/ws-manager/pkg/manager/create.go#L700).
Setting it to 0 will only add more readiness failure logs. At least set this to one, please.

Reply:
In mk1 we do not use it to declare the workspace as Running though. If you object I instead propose we use ContentInit condition.